qa/tasks/cephfs/test_nfs.py

```diff
@@ -369,6 +371,64 @@ def _test_data_read_write(self, pseudo_path, port, ip):
         except CommandFailedError as e:
             self.fail(f"expected read/write of a file to be successful but failed with {e.exitstatus}")
 
+    def _mnt_nfs(self, pseudo_path, port, ip):
+        '''
+        Mount created export
+        :param pseudo_path: It is the pseudo root name
+        :param port: Port of deployed nfs cluster
+        :param ip: IP of deployed nfs cluster
+        '''
+        tries = 3
+        while True:
```

Comment:
A better contender here would be `contextutil.safe_while()`. It does exactly this but would keep this code concise.

Reply:
I had just reused the exisiting piece of code. I will look into it

Reply:
I will fix this as part of https://tracker.ceph.com/issues/68870